person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'Unknown', _parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    of_age? || parent_permission
```

Comment:
- [ ] Kindly use the required variable as requested by the reviewer.

  ```
  of_age? || @parent_permission
  ```

Reply:
@DammyShittu, the previous reviewer mentioned that,  The parent permission here is not an instance variable so it would cause errors. You should use the @parent_permission instance. And we don't need to repeat ourselves by checking if it is true or false, since @parent_permission is already boolean.
Thanks.

Reply:
Using the `parent_permission` like this without the `@` means that we have to supply an external variable `parent_permission` different from the one local to this class, `@parent_permission`. That does not exist (and should not) so the method will cause error as rightly pointed out by @TedLivist

Reply:
@DammyShittu, thanks a lot